.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Overview
+
+< A brief description of the pull request >
+
+---
+
+| Q             | A
+| ------------- | ---
+| Bug fix?      | yes/no
+| New feature?  | yes/no
+| BC breaks?    | yes/no
+| Deprecations? | yes/no
+| Tests pass?   | yes/no
+
+---
+
+## Before
+
+< The current status >
+
+## After
+
+< What will be changed >
+
+## Screenshots
+
+< If required. [LICEcap](http://www.cockos.com/licecap/) for Mac or SilentCast for Linux allow you to great GIFs from screen capture >
+
+## Comments
```

Comment:
I think it's a good idea to keep the overview non-technical, but for before and after or comments we can be technical. Can you maybe give an example of where we would need a separate technical notes section?